src/content/events/davos.md

```diff
@@ -13,12 +14,145 @@ location:
 start-date: 2025-01-20T09:00:00.000Z
 end-date: 2025-01-23T17:00:00.000Z
 image:
-  src: /assets/images/davos-penthouse-25.webp
+  src: /assets/images/ff-at-davos-25.webp
+program:
+  events:
+    - title: The Filecoin Penthouse. Hosted by Filecoin Foundation
+      start-date: 2025-01-20T09:00:00.000Z
+      end-date: 2025-01-23T17:30:00.000Z
+      location: Davos, Switzerland
+      description:
+        "The Filecoin Penthouse is an exclusive lounge in the heart of
+        Davos, designed for networking, casual meetings, and relaxing with
+        coffee while enjoying breathtaking views. A private meeting room
+        (seating up to 10 people) is available upon request. Located in WEF’s
+        Secure Zone, the venue requires a WEF Hotel Badge or other official WEF
+        event badge or credential for entry. Book a private meeting room:
+        events@fil.org"
+      external-link: https://cvent.me/xbZ2w2
+    - title:
+        "With Honor Action Reception. With Special Guests: Members of the U.S.
+        Congressional Delegation. Hosted by Filecoin Foundation, With Honor
+        Action"
+      start-date: 2025-01-22T20:30:00.000Z
+      description:
+        With Honor Action, in partnership with Filecoin Foundation, is
+        hosting the 4th Annual Reception for the bipartisan U.S. Congressional
+        Delegation to the World Economic Forum. This invitation-only event will
+        occur on Wednesday night. Space is limited, and attendance is by prior
+        confirmation only.
+      location: "Davos, Switzerland"
+    - title:
+        Policy in Draft Reception. Hosted by Filecoin Foundation, Hedera, and
+        GBBC.
+      start-date: 2025-01-22T17:00:00.000Z
+      location: Davos, Switzerland
+      description: The Policy in Draft Reception, hosted by Filecoin Foundation,
+        Hedera, and GBBC, will bring together senior industry leaders and
+        policymakers from around the globe for an engaging dialogue on the
+        future of technology and governance. This event will explore key policy
+        priorities, the evolving landscape of blockchain use cases, and
+        strategies for meaningful collaboration between industry and government
+        as we look toward 2025 and beyond.
+      external-link: https://cvent.me/Z5Gk11
+  title: Featured Events
+  kicker: Find us around Davos
+schedule:
+  title: Speaking Engagements
+  days:
+    - events:
+        - title: Marta Belcher at GBBC’s 8th Annual Blockchain Central Davos
+          location: Grandhotel Belvedere | Promenade 89, 7270 Davos, Switzerland
+          tag: Registration Needed
+          start: 2025-01-20T00:00:00.000Z
```

Comment:
This says midnight, maybe let's use noon or something generic if we don't know the time yet?

Reply:
I'm ok with that - people will not see this until closer to the date... ideally no time would be best, but ok if we need to put something!